### PR TITLE
Support for NFS problem added 

### PR DESCRIPTION
**Pull request by DESHRAJ**
While setting up Treeherder-service, executing the vagrant up, an error is encountered if anyone don't have nfs-kernel-server installed in their system. I am adding the support for this. I also encountered the same error. 
Please review this.  :-)

### CODE REVIEW

**Comment by tojonmz** [comment]
Thanks for looking into this @DESHRAJ! If your can update the commit and the title of the PR that would be helpful, in the form:

`Bug nnnnn - Summary of the bug`

You can see that general format observed here
https://github.com/mozilla/treeherder-service/pulls

If you're looking to see if there's an existing one, I wonder if this [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1056049) is what you saw? If not, here's a [query](https://bugzilla.mozilla.org/buglist.cgi?list_id=12015308&short_desc=vagrant&resolution=---&query_format=advanced&short_desc_type=allwordssubstr&product=Tree%20Management) also, for all Treeherder bugs with 'vagrant' in the summary, in case it might be in that list.

If your bug is different though, feel free to open a new one, Product=TreeManagement, Component=Treeherder:Docs&Development, and we'll show you how to connect the bug and this PR for review. You can also check in with @edmorley for a suitable component if needed.

I couldn't tell from the backscroll in IRC if it had been entered or not.

Thanks for looking into this!

File: docs/installation.rst [hunk 1]
```diff
@@ -22,6 +22,13 @@ Setting up Vagrant
   .. code-block:: bash
 
      >vagrant up
+  
+  Note: If you encouter error showing "Your machine doesn't supported NFS, or there is not an
+  adapter to enable NFS on this machine for Vagrant",then install ``nfs-kernel-server`` using the command:
```
Comment (tojonmz) [inline docs/installation.rst]: If we add these and other messages (see below) I wonder if we might want to present the error in a code block as well. I think the error above got a bit altered during writing, eg. *'doesn't supported'* above. Here's a link to all the errors (I think) for reference. I defer to @edmorley how best to present them.
https://github.com/edx/edx-platform/wiki/Simplified-install-with-vagrant

**Comment by edmorley** [comment]
I'm happy for this to land with a corresponding bug :-)
If you could just squash the two commits and tweak the commit message to describe what was changed, we should be good to go here (eg "docs: add instructions for installing nfs").

**Comment by DESHRAJ** [comment]
I have closed this request. I will make a fresh pull request and everything will be updated this time.  :)

**Comment by edmorley** [comment]
You don't need to create a new PR (it's preferred not to, otherwise it creates lots of duplicate PRs). Please update locally an |git push -f| to the existing branch. I'll then reopen this one (the button is greyed out until there are new commits here).

**Comment by DESHRAJ** [comment]
@edmorley, I have made the commit again, now how to reopen this pull request ?

**Pull request closed without merging**